src/main/java/fr/alasdiablo/janoeo/world/gen/OverworldOreGenerator.java

```diff
@@ -182,7 +116,7 @@ public void startWorldGeneration(Biome biome) {
                         GenerationStage.Decoration.UNDERGROUND_ORES
                 );
             }
-            if (GLOBAL_CONFIG.EXTRA_ORE_GEN.get()) {
+            if (GLOBAL_CONFIG.EXTRA_ORE_GEN.get() && false) {
```

Comment:
Why ?

Reply:
Was just to get the mod working as these ore values were not setup yet using the new method and the mod would crash on load. This pull request was meant as a proof of concept not an actually mergeable request.